docs/hub/datasets-libraries.md

```diff
@@ -0,0 +1,16 @@
+# Libraries
+
+The Dataset Hub has support for several libraries in the Open Source ecosystem.
+Thanks to the `huggingface_hub` Python library, it's easy to enable sharing your datasets on the Hub.
+The Hub supports many libraries, and we're working on expanding this support!
```

Comment:
I think it's redundant with the first line

Reply:
removed it